test-integ/go.mod

```diff
@@ -0,0 +1,246 @@
+module github.com/hashicorp/consul/test-integ
```

Comment:
What is this test-integ module for ? Also don't love a hypen in a package name https://go.dev/blog/package-names 

Finally this is  under consul which means it's a integration test for all of consul which I don't think you were intending to make?  Let me know what you think.

Reply:
Oh thanks - I need to remove this. I just cherry-picked the original commit. It's a bit weird but `main` has these files, but not this release branch.

I will remove these